src/algorithms/explicit_rk.jl

```diff
@@ -113,6 +113,78 @@ function RKM(stage_limiter!, step_limiter! = trivial_limiter!)
     RKM(stage_limiter!, step_limiter!, False())
 end
 
+@doc explicit_rk_docstring("4 stage Pseudo-Symplectic Explicit RK method.", "3p5q(4)",
+    references = "@article{Aubry1998,
+    author = {A. Aubry and P. Chartier},
+    journal = {BIT Numer. Math.},
+    title =  {Pseudo-symplectic {R}unge-{K}utta methods},
+    year = {1998},
+    },
+    @article{Capuano2017,
+    title = {Explicit {R}unge–{K}utta schemes for incompressible flow with improved energy-conservation properties},
+    journal = {J. Comput. Phys.},
+    year = {2017},
+    author = {F. Capuano and G. Coppola and L. Rández and L. {de Luca}},}")
+Base.@kwdef struct PSRK3p5q4{StageLimiter, StepLimiter, Thread} <: OrdinaryDiffEqAlgorithm
+    stage_limiter!::StageLimiter = trivial_limiter!
+    step_limiter!::StepLimiter = trivial_limiter!
+    thread::Thread = False()
+end
+# for backwards compatibility
+function PSRK3p5q4(stage_limiter!, step_limiter! = trivial_limiter!)
+    PSRK3p5q4(stage_limiter!, step_limiter!, False())
+end
```

Comment:
```suggestion
```
Since the method is new, we don't need the additional constructors for backwards compatibility.